assets/package-infos.json

```diff
@@ -7605,7 +7605,7 @@
                 "IsMaintainer": false,
                 "LastName": "Felsch",
                 "Place": "Aachen",
-                "WWWHome": "http://www.math.rwth-aachen.de/LDFM/homes/Volkmar.Felsch/"
+                "WWWHome": "http://www.math.rwth-aachen.de/homes/Volkmar.Felsch/"
```

Comment:
This needs to be fixed in the PackageInfo.g of the `itc` package and be put into a release, otherwise the next GAP update will reset this.